src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -169,11 +169,16 @@ export class TxSender {
       return
     }
 
-    const contractErrorMessage = await getContractErrorMessage({
-      safeInstance,
-      from,
-      data: executeData,
-    })
+    let contractErrorMessage
+    try {
+      contractErrorMessage = await getContractErrorMessage({
+        safeInstance,
+        from,
+        data: executeData,
+      })
+    } catch (err) {
+      notifications.showOnError(err, err.message)
+    }
```

Comment:
Then we would add the error log & notification always on the same way.

```suggestion
    let contractError = await getContractErrorMessage({
        safeInstance,
        from,
        data: executeData,
      })
      
      if (contractError) {
        logError(contractError.code, contractError.message)
        notifications.showOnError(err, contractError.message)
      }
```